cypress/fixtures/extensions/extension-foo/filters.js

```diff
@@ -0,0 +1,5 @@
+const addFilter = require('govuk-prototype-kit').nunjucks.addFilter
+const getFilter = require('govuk-prototype-kit').nunjucks.getFilter
```

Comment:
This could be more concise

```suggestion
const { addFilter, getFilter } = require('govuk-prototype-kit').nunjucks
```